server/src/main/java/org/elasticsearch/index/fielddata/plain/PagedBytesIndexFieldData.java

```diff
@@ -9,8 +9,8 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.apache.lucene.codecs.blocktree.FieldReader;
-import org.apache.lucene.codecs.blocktree.Stats;
+import org.apache.lucene.codecs.lucene90.blocktree.FieldReader;
+import org.apache.lucene.codecs.lucene90.blocktree.Stats;
```

Comment:
Now that blocktree is versioned and that fielddata isn't used much anymore, I'm tempted to remove the optimization that uses these two classes. Otherwise we're going to have a testing gap with pre-Lucene 9 indices, for which this optimization doesn't work, and that our tests don't exercise much.

Reply:
Removed in 01b3050c02a09b5d684277ff49c2684a0c5fdf97